cookbook/remote_access/inspecting_executions.py

```diff
@@ -21,4 +24,34 @@
 
     "outputUri": "s3://my-s3-bucket/metadata/propeller/flytesnacks-development-<execid>/n0/data/0/outputs.pb"
 
+
+FlyteRemote:
+========================
+
+With FlyteRemote, you can fetch the inputs and outputs of execution and inspect them.
+
+.. code-block:: python
+
+    from flytekit.remote import FlyteRemote
+
+    # FlyteRemote object is the main entrypoint to API
+    remote = FlyteRemote(
+        config=Config.for_endpoint(endpoint="flyte.example.net"),
+        default_project="flytesnacks",
+        default_domain="development",
+    )
+    
+    execution = remote.fetch_execution(
+        name="fb22e306a0d91e1c6000", project="flytesnacks", domain="development"
+    )
+
+    # Inspecting execution
+    # The 'inputs' and 'outputs' correspond to the top-level execution or the workflow itself.
+    input_keys = execution.inputs.keys()
+    output_keys = execution.outputs.keys()
+
+    # You can use FlyteRemote.sync() to to sync the entity object's state with the remote state during the execution run
+    synced_execution = remote.sync(execution, sync_nodes=True)
+    node_keys = synced_execution.node_executions.keys()
```

Comment:
Can you move this to the end, like in https://docs.flyte.org/projects/flytekit/en/latest/design/control_plane.html#retrieving-inspecting-executions? Also show how to fetch output of a specific node execution.